amp/WEB-INF/src/org/dgfoundation/amp/onepager/components/QuarterInformationPanel.java

```diff
@@ -33,7 +33,7 @@
  * @since 04 Feb 2013
  */
 public class QuarterInformationPanel extends AmpFieldPanel {
-    private static final JavaScriptResourceReference JS_FILE = new JavaScriptResourceReference(
+    public static final JavaScriptResourceReference JS_FILE = new JavaScriptResourceReference(
             QuarterInformationPanel.class, "QuarterInformationPanel.js");
```

Comment:
since the file has already been added on AmpSubsectionFeatureFundingPanel#renderHead I think we can remove that line from here